src-tauri/src/main.rs

```diff
@@ -100,6 +100,8 @@ fn main() {
     windows::winreg_get,
     windows::winreg_set,
     windows::winreg_delete,
+    get_this_binary,
```

Comment:
@nazar-pc I found that this command was not compiled on windows unless I add it here.

Reply:
Yeah, that is correct after I read the implementation in Tauri, thanks!